presto-hive/src/test/java/com/facebook/presto/hive/TestHiveRecoverableGroupedExecution.java

```diff
@@ -332,6 +347,28 @@ public void testUnionAll(int writerConcurrency)
                         "DROP TABLE IF EXISTS test_failure"));
     }
 
+    @Test
+    public void testCountOnUnbucketedTable()
+            throws Exception
+    {
+        queryRunner = createQueryRunner(true);
+        testRecoverableGroupedExecution(
```

Comment:
Now we are not only testing `RecoverableGroupedExecution` but also stage retry right? -- So maybe just `testRecoverableExecution` ? :)